docs/asm-gke-terraform/main.tf

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
+# Copyright 2023 Google LLC
```

Comment:
We should not update the copyright year (go/copyright).

We can either retain the earliest date (`2022` in this case) or have a range of dates that we keep up-to-date (`2022-2023`). The former is easier / means less maintenance / doesn't need to be updated every year.

In either case, `2023` is not a valid option. :)

Reply:
Thanks. TIL. I changed it back to 2022.